### PR TITLE
docs: add initial doc for handling CVEs

### PR DESCRIPTION
**Pull request by raghavendra-talur**
Adding a doc to describe the methods to handle a CVEs. We can extend this later.

### CODE REVIEW

File: docs/dev/cve.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+security flaw that's been assigned a CVE ID number.
+
+## CVEs filed on dependencies
+
+### Checking if a CVE applies to Ramen
+
+**Note**: Ramen has two go modules, one for the main project and another for
+the api. You need to perform the check below for both the modules.
+
+When a CVE is filed on a direct or a indirect dependency of Ramen, you will
```
Comment (ELENAGER) [inline docs/dev/cve.md:16]: Typo - an indirect

**Review by ELENAGER — APPROVED** [review]
LGTM

File: docs/dev/cve.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    ```
+    $ go mod why -m github.com/open-telemetry/opentelemetry-go-contrib
+    # github.com/open-telemetry/opentelemetry-go-contrib
+    (main module does not need module github.com/open-telemetry/opentelemetry-go-contrib)
+    ```
+
+## Fixing the CVEs by updating the packages
+
+1. When a module is a direct dependency, then using `go get` to update the
+   version of the module to the one that fixes the CVE is sufficient.
```
Comment (nirs) [inline docs/dev/cve.md:66]: Adding an example command would be nice.

File: docs/dev/cve.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+1. When a module is an indirect dependency, you will have to determine if the
+   module which brings in the transitive dependency has a fix for the CVE. If
+   it does, you can update the version of the module to the one that fixes the
+   CVE using `go get`. If it doesn't, you will have to wait for the module to
+   update the version of the transitive dependency or you can use a replace
+   directive in the go.mod file to update the version of the transitive
+   dependency.
```
Comment (nirs) [inline docs/dev/cve.md:74]: Adding an example commands for such case can help.

**Review by nirs — APPROVED** [review]
Thanks for documenting this, added notes about additional info that can help to fix the issues.

**Review by ShyamsundarR — APPROVED** [review]
LGTM, will wait in case you are adding examples.

**Pull request merged**